INSTALL_GUIDE.md

```diff
@@ -161,7 +163,34 @@ Inside this file, paste the copied URL and Key from the website and save.
 
 ![alt text](doc-images/image3.png)
 
-## 6. IDE / Text Editors
+## 6. Windows Path Modifications (Optional)
+
+If running LDAR-Sim on windows, you may encounter errors related file path length. LDAR-Sim V4 is design to provide warnings when file paths begin to become to long. To resolve this problem, users may enable windows long file paths or move LDAR-Sim closer to the C drive in folder organization. To use git with longer filepaths, the long file paths must  also be added to the git settings.
+
+### Enabling Long File Paths in Windows (Example is from Windows 11)
+
+1. Open the windows run dialog with Windows Key + R at the same time
+   ![alt text](doc-images/Windows%20Run%20Dialog.png)
+
+2. Type gpedit.msc into the run dialog and hit ok. You will see the group policy editor open up
+   ![alt text](doc-images/group%20policy%20editor.png)
+3. Navigate to **Local Computer Policy** > **Computer Configuration** > **Administrative Templates** **System** > **Filesystem**
+   ![alt text](doc-images/Group%20Policy%20at%20Filesystem.png)
+4. Double Click on "Enable Win32 long paths
+   ![alt text](doc-images/group%20policy%20enable%20long%20paths.png)
+5. Select Enable, click Apply and then OK and restart the computer
+   ![alt text](doc-images/Enable%20Win%2032%20long%20paths.png)
+
```

Comment:
Nit: should there be a warning written here that altering this may cause issues with older applications on windows that cannot handle long pathways?

Reply:
Uhhh - Yeah probably ...